File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    const int MAX_COUNT = 2;
    int removeDuplicates(vector<int>& nums) {
        if(nums.size() <= 2) return nums.size();
        int j = 0;
        for (int num : nums) {
            if (j < 2 || num != nums[j - 2]) {
                nums[j++] = num;
            }
        }
        return j;
    }
};

File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if(nums.length <= 2) return nums.length;
    let j = 0;
    for(let i = 0 ; i < nums.length; ++i){
        if(j < 2 || nums[i] !== nums[j - 2]){
            nums[j++] = nums[i];
        }
    }
    return j;
};